src/app/courses/components/new-corse-date/new-corse-date.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="input-wrapper">
```

Comment:
Same comments for this component.
The name of the component should be abstract. CSS classes should be appropriate.